### PR TITLE
[Linux] Fix Lintian warnings

### PR DESCRIPTION
**Pull request by basilgello**
## Description

Fixes Lintian warning.

## Motivation and Context

Trying to make Debian pakcage lintian-clean :+1: 

## How Has This Been Tested?

Build; run lintian on resulting deb-files.

## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [x] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [x] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [x] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [x] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [x] All new and existing tests passed

### CODE REVIEW

**Comment by basilgello** [comment]
@phunkyfish

**Comment by mapreri** [comment]
More references in https://specifications.freedesktop.org/desktop-entry-spec/latest/ar01s06.html and https://specifications.freedesktop.org/desktop-entry-spec/latest/apes04.html

**Comment by basilgello** [comment]
I am going to add several other commits fixing Lintian warnings. All them are safe to include before 19.0 beta1 as they are cosmetic changes.

**Review by garbear — APPROVED** [review]
No risky code changes; Python scripts aren't executed from command line, so the shebang doesn't matter (what would this do on Android or iOS?); and the desktop shortcut changes are welcome.

File: tools/darwin/Support/GenerateMissingImages-tvos.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
```
Comment (fuzzard) [inline tools/darwin/Support/GenerateMissingImages-tvos.py:1]: Do not change this. Does not work on OSX Catalina

```
Support % ./GenerateMissingImages-tvos.py ../../../xbmc/platform/darwin/tvos/Assets.xcassets Assets LaunchImage
/Users/brent/osx-master/media/splash.jpg
  /Users/brent/osx-master/xbmc/platform/darwin/tvos/Assets.xcassets/LaunchImage.launchimage/splash@2x.jpg
/Users/brent/osx-master/xbmc/platform/darwin/tvos/Assets.xcassets/Assets.brandassets/topshelf_wide.imageset/image.png
  /Users/brent/osx-master/xbmc/platform/darwin/tvos/Assets.xcassets/Assets.brandassets/topshelf_wide.imageset/image@2x.png
Traceback (most recent call last):
  File "./GenerateMissingImages-tvos.py", line 28, in <module>
    for i in xrange(1, 5):
NameError: name 'xrange' is not defined
```

**Comment by fuzzard** [comment]
Have any of the py changes been tested running on py3? 
There are changes in the language, and most look like they were definitely built in the py2 era. Some may need language changes, and not just s shebang change.

**Comment by basilgello** [comment]
I do use kodi-send sinve I migrated to 19.0 in April. As Python 2 gets deprecated anyway, I think I will fix the scripts to work with Python 3 because niw it is an ideal moment for it.
-- 
Vasyl Gello
==================================================
Certified SolidWorks Expert

Mob.:+380 (98) 465 66 77

E-Mail: vasek.gello@gmail.com

Skype: vasek.gello
==================================================
호랑이는 죽어서 가죽을 남기고 사람은 죽어서 이름을 남긴다

File: tools/darwin/Support/GenerateMissingImages-tvos.py (hunk 1 above)
Comment (basilgello) [reply tools/darwin/Support/GenerateMissingImages-tvos.py:1]: Thanks for testing this! Yes xrange changes is trivial - I will walk through the files and fix the py2-specific issues.

File: docs/manpages/TexturePacker.1 [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+A common misconception is that TexturPacker will just compress the images into a single file. This if far from the truth, so don't be surprised if the Textures.xbt file is much larger than the total size of all the individual images.
+
+The optional -dupecheck option is useful if you have included the same image multiple times in your media folder. For example, if your skin includes a lot of studio logos, most likely a lot of them are included multiple times but with a slightly different filename. The '-dupecheck' option will make sure each unique image is included only once in the Textures.xbt file, thus keeping the size of it as small as possible.
+.SH OPTIONS
+.TP
+.BR \-dupecheck
+Check for image duplicates first
+.TP
+.BR \-inout
```
Comment (garbear) [inline docs/manpages/TexturePacker.1:20]: `input` I believe

Comment (basilgello) [reply docs/manpages/TexturePacker.1]: Fixed.

File: tools/darwin/Support/GenerateMissingImages-tvos.py (hunk 1 above)
Comment (basilgello) [reply tools/darwin/Support/GenerateMissingImages-tvos.py:1]: I fixed this issue. Please test it when you can.

**Comment by basilgello** [comment]
Now I am going to run-test tge affected python scripts once more and add another commit fixing manpage for `kodi.bin`

**Comment by basilgello** [comment]
OK, this PR is ready for review.

**Review by garbear — APPROVED** [review]
Some code changes have been added, but they're all in the `tools/` folder so I have no concerns.

**Comment by basilgello** [comment]
@fuzzard can you please verify the tvOS helper works as expected? The examples except C# have been verified by me. PS3 scripts also start and try pairing with BT joystick but I dont have one.

File: tools/darwin/Support/GenerateMissingImages-tvos.py (hunk 1 above)
Comment (fuzzard) [reply tools/darwin/Support/GenerateMissingImages-tvos.py:1]: this change is ok now.

File: addons/service.xbmc.versioncheck/resources/lib/version_check/service.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
```
Comment (fuzzard) [inline addons/service.xbmc.versioncheck/resources/lib/version_check/service.py:1]: This change has to be done upstream https://github.com/XBMC-Addons/service.xbmc.versioncheck

File: addons/service.xbmc.versioncheck/resources/lib/version_check/viewer.py [hunk 4]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
```
Comment (fuzzard) [inline addons/service.xbmc.versioncheck/resources/lib/version_check/viewer.py:1]: same again https://github.com/XBMC-Addons/service.xbmc.versioncheck

**Review by fuzzard — COMMENTED** [review]
the demo metadata changes are ok in tree, but the service.xbmc.versioncheck changes needs to be sent upstream, otherwise they will just be wiped next time its synced.

**Comment by wsnipex** [comment]
jenkins build this please

**Comment by DaveTBlake** [comment]
Have the versioncheck changes been submitted upstream? And if so do they need to be here too since repo sync will overwrite.

**Comment by basilgello** [comment]
Not yet, sorry @DaveTBlake! Will do tonight!

**Comment by DaveTBlake** [comment]
Going to bump to next release to ensure so that version check changes get done in right place. Looking to merge this soonest though @basilgello

**Comment by basilgello** [comment]
@DaveTBlake @phunkyfish

File: addons/service.xbmc.versioncheck/resources/lib/version_check/service.py (hunk 3 above)
Comment (enen92) [reply addons/service.xbmc.versioncheck/resources/lib/version_check/service.py:1]: Done and merged

**Review by phunkyfish — APPROVED** [review]
Thanks for upstreaming. Looks good now.

**Comment by howie-f** [comment]
jenkins build this please

**Comment by howie-f** [comment]
jenkins build this please

**Comment by howie-f** [comment]
jenkins build this please

**Comment by howie-f** [comment]
jenkins build this please

**Pull request merged**